app/models/notification.rb

```diff
@@ -15,11 +17,17 @@ class Notification < ApplicationRecord
   # -- Scopes ---------------------------------------------------------------
   scope :unread,  -> { where(read_at: nil) }
   scope :read,    -> { where.not(read_at: nil) }
+  scope :newest,  -> { order(created_at: :desc) }
 
   # -- Class Methods --------------------------------------------------------
 
   # -- Instance Methods -----------------------------------------------------
   def read?
     self.read_at
   end
+
+  def unread?
```

Comment:
you can probably remove this and update the only place in the view where that's used with a call to `read?`